src/components/ChoiceList/tests/ChoiceList.test.tsx

```diff
@@ -3,10 +3,10 @@ import {mountWithApp} from 'test-utilities';
 // eslint-disable-next-line no-restricted-imports
 import {mountWithAppProvider, ReactWrapper} from 'test-utilities/legacy';
 import {RadioButton, Checkbox, InlineError, errorTextID} from 'components';
-import {ChoiceList, ChoiceDescriptor} from '../ChoiceList';
+import {ChoiceList, ChoiceListProps} from '../ChoiceList';
 
 describe('<ChoiceList />', () => {
-  let choices: ChoiceDescriptor[];
+  let choices: ChoiceListProps['choices'];
```

Comment:
Where possible we should look at the values that live on the Props interface  instead of exposing the interfaces the Props use